cmake/Dependencies.common.cmake

```diff
@@ -275,7 +275,7 @@ endif()
 set(DALI_INSTALL_REQUIRES_NVIMGCODEC "")
 if(BUILD_NVIMAGECODEC)
   set(NVIMGCODEC_MIN_VERSION "0.3.0")
-  set(NVIMGCODEC_MAX_VERSION "0.4.0")
+  set(NVIMGCODEC_MAX_VERSION "0.4.1")
```

Comment:
This has been done in https://github.com/NVIDIA/DALI/pull/5576/. I would not touch this here yet (just in case).